engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -319,19 +304,16 @@ private void renderShadowMap() {
     public void renderWorldReflection() {
         PerformanceMonitor.startActivity("Render World (Reflection)");
 
-        DefaultRenderingProcess.getInstance().beginRenderReflectedScene();
-        GL11.glCullFace(GL11.GL_FRONT);
+        LwjglRenderingProcess.getInstance().beginRenderReflectedScene();
         playerCamera.setReflected(true);
 
-        playerCamera.lookThroughNormalized();
+        playerCamera.lookThroughNormalized(); // we don't want the reflected scene to be bobbing or moving with the player
         backdropRenderer.render(playerCamera);
         playerCamera.lookThrough();
 
         chunkShader.activateFeature(ShaderProgramFeature.FEATURE_USE_FORWARD_LIGHTING);
 
         if (renderingConfig.isReflectiveWater()) {
-            GL11.glEnable(GL_LIGHT0);
```

Comment:
Don't worry about it too much, it is more just me mentioning random memories :-)